### ISSUE TITLE
Is this package still under maintenance?

### ISSUE THREAD

**Issue by Kamomede** [LiWenHui96/flutter_file_view#35]
When I load flutter_file_view in pubspec.yaml, it obviously prompts that the dio package in flutter_file_view is too low.

Because flutter_file_view >=2.0.3 depends on dio ^4.0.0 and wolkezoo_app depends on dio ^5.3.3, flutter_file_view >=2.0.3 is forbidden.
So, because wolkezoo_app depends on flutter_file_view ^2.2.1, version solving failed.

**Comment by yanashenyang** [comment]
same
Because flutter_file_view 2.2.1 depends on dio ^4.0.0 and no versions of flutter_file_view match >2.2.1 <3.0.0, flutter_file_view ^2.2.1
  requires dio ^4.0.0.

**Comment by Kamomede** [comment]
Yes, that's right, but currently the dio package has been maintained to 5.4.1. Do you have any plans to update the dio version?
